server/channels/api4/user_test.go

```diff
@@ -7848,3 +7848,76 @@ func TestLoginWithDesktopToken(t *testing.T) {
 		assert.Len(t, sessions, 0)
 	})
 }
+
+func TestSetExtraSessionProps(t *testing.T) {
+	th := Setup(t).InitBasic()
+	defer th.TearDown()
+
+	client := th.CreateClient()
+	th.LoginBasicWithClient(client)
+
+	resetSession := func(session *model.Session) {
+		session.AddProp(model.SessionPropDeviceNotificationDisabled, "")
+		session.AddProp(model.SessionPropMobileVersion, "")
+		th.Server.Store().Session().UpdateProps(session)
+		th.Server.Platform().AddSessionToCache(session)
+	}
+
+	t.Run("No props will return ok and no changes in the session", func(t *testing.T) {
+		session, _ := th.App.GetSession(client.AuthToken)
+		defer resetSession(session)
+		res, err := client.SetExtraSessionProps(context.Background(), map[string]string{})
+		updatedSession, _ := th.App.GetSession(client.AuthToken)
+		storeSession, _ := th.Server.Store().Session().Get(th.Context, session.Id)
+		assert.Equal(t, http.StatusOK, res.StatusCode)
+		assert.NoError(t, err)
```

Comment:
Can we move the `NoError` check next to its declaration? This happens in all subsequent tests as well.
```suggestion
		res, err := client.SetExtraSessionProps(context.Background(), map[string]string{})
		assert.NoError(t, err)
		updatedSession, _ := th.App.GetSession(client.AuthToken)
		storeSession, _ := th.Server.Store().Session().Get(th.Context, session.Id)
		assert.Equal(t, http.StatusOK, res.StatusCode)
```